starter-code/public/scripts/controllers/aboutController.js

```diff
@@ -7,9 +7,14 @@ var app = app || {};
   aboutController.index = () => {
     $('#about').show().siblings().hide(); // REVIEW: We have a slight refactor in selectors here, which has reduced the amount of code from the last lab.
 
-    // TODO: Call a function to load all the data.
+    // DONE: Call a function to load all the data.
     // Pass a view function as a callback, so the view will render after the data is loaded.
 
+    module.repos.requestRepos(
```

Comment:
This does not need to be in a function. You can just call requestRepos with .index as the callback. Index does not take a parameter, so you should not have one ther.e